cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -177,33 +179,28 @@ def test_delete_all_snapshots(create_vm, provider):
              message="Waiting for second snapshot to disappear")
 
 
+@pytest.mark.meta(blockers=[BZ(1805803, forced_streams=["5.10"])], automates=[1805803])
```

Comment:
@prichard77 I think this should prevent the test to be ran with the failing configuration.  Did you try find out why it is not effective?

Reply:
I just thought that the blocker line was not effective when you specifically entered it in the PRT section of the PR. It shouldn't run when we do a full collection and run of tests. I am actually glad it runs here so we can check out the failures and continue with review.

Reply:
But the blocker was in the wrong place. I have put it in the correct place now. We can look at the old runs but it will get skipped in 5.10. BZ 1805803 is in 5.11 but it does not fix the issue.I have put it back to on_Dev. Thanks for pointing this issue out, Jaroslav.